src/main/java/lotto/domain/lotto/LottoNumbers.java

```diff
@@ -3,44 +3,53 @@
 import java.util.*;
 
 public class LottoNumbers {
-
     public static final int LOTTO_NUMBER_SIZE = 6;
-    private static final Numbers numbers = Numbers.create();
-    private List<Integer> lottoNumbers;
+    private static final NumberGenerator NUMBER_GENERATOR = NumberGenerator.create();
+    private List<LottoNumber> lottoNumbers;
```

Comment:
가독성을 높이기 위해서, 상수와 변수 사이에 공백을 하나 넣는건 어떨까요??
```suggestion

    private List<LottoNumber> lottoNumbers;
```